### ISSUE TITLE
Double-click of image in Windows fails if application is already open

### ISSUE THREAD

**Issue by itsayellow** [itsayellow/marcam#109]
Windows tries to start another Marcam executable, and this fails because the new Marcam executable can't move the old log files to make a new log file:
```
UNCAUGHT FATAL ERROR
Traceback (most recent call last):
  File "marcam/marcam.py", line 2298, in <module>
    STATUS = main(sys.argv)
  File "marcam/marcam.py", line 2280, in main
    logging_setup(log_level)
  File "C:\Users\mclapp\git\marcam\marcam\common.py", line 43, in func_wrapper
    return_vals = func(*args, **kwargs)
  File "marcam/marcam.py", line 131, in logging_setup
    os.replace(fname, fname_plus_1)
PermissionError: [WinError 32] The process cannot access the file because it is
being used by another process: 'C:\\Users\\mclapp\\AppData\\Local\\Marcam\\Logs\
\marcam.log' -> 'C:\\Users\\mclapp\\AppData\\Local\\Marcam\\Logs\\marcam.log.1'
```

Need to:
* Find a way on Windows for double-clicking file to send it to application already open
* Allow new instance of marcam to run even if log file is locked

**Comment by itsayellow** [comment]
Clean ways:
* Using sockets.  Keep an open socket to receive open file requests, then new app sees old app running and sends filepath to open socket of orig. program, then new program shuts down.
* Using Windows message passing: WM_COPYDATA
  * https://www.makemolten.com/articles/sending-wmcopydata-in-python-with-ctypes/
  * https://stackoverflow.com/questions/19886633/sending-wm-copydata-with-python-3/20036594#20036594
  * https://docs.microsoft.com/en-us/windows/desktop/dataxchg/using-data-copy

**Comment by itsayellow** [comment]
Using Windows named pipes, (winpipe.py) we can send filenames to be opened to the main instance from any other instances opened by file double-click.  

Any other instances are only run to send filenames to the main instance, and then shut down.

Fixed with commit a71c56e.
